src/js/components/NodesGridDials.js

```diff
@@ -144,7 +144,9 @@ var NodesGridDials = React.createClass({
   },
 
   getDials: function () {
-    return _.map(this.props.hosts, function (node) {
+    let hosts = this.props.hosts || [];
+
+    return hosts.map(function (node) {
```

Comment:
`map` doesn't take a 2nd argument. this whole block of code should not be working right now (i haven't tested)